packages/react-form-renderer/src/components/form-renderer.js

```diff
@@ -22,15 +22,17 @@ const FormRenderer = ({
   subscription,
   clearedValue,
   schema,
-  validatorMapper
+  validatorMapper,
+  actionMapper
 }) => {
   let schemaError;
 
   const validatorMapperMerged = { ...defaultValidatorMapper, ...validatorMapper };
 
   try {
     const validatorTypes = Object.keys(validatorMapperMerged);
-    defaultSchemaValidator(schema, formFieldsMapper, validatorTypes);
+    const actionTypes = actionMapper ? Object.keys(actionMapper) : {};
```

Comment:
` Object.keys(actionMapper)` will return `true` even on an empty object. Is that desired?

Reply:
Furthermore, sometimes the `actionTypes` is an array and sometimes object.

Reply:
Yeah, it won't do any harm.

Reply:
> Furthermore, sometimes the actionTypes is an array and sometimes object.

What you think? actionTypes should be array with names of the actions used to validate the schema

Reply:
SO maybe have the fallback value `[]` instead of `{}`?

Reply:
oh :hankey:  you are right :D